thrift/src/main/thrift/contentatom.thrift

```diff
@@ -2,17 +2,16 @@ namespace java com.gu.contentatom.thrift
 #@namespace scala com.gu.contentatom.thrift
 
 include "atoms/quiz.thrift"
-include "atoms/viewpoints.thrift"
 include "atoms/media.thrift"
 include "shared.thrift"
 
 typedef string ContentAtomID
 
-enum AtomType { QUIZ, VIEWPOINTS, MEDIA }
+enum AtomType { QUIZ, MEDIA }
```

Comment:
It's probably safest to specify the enum ordinals so that Media's ordinal doesn't change:

```
enum AtomType {
  QUIZ = 0,
  MEDIA = 2
}